.github/workflows/e2e-arm64.yaml

```diff
@@ -1,11 +1,6 @@
 ---
 name: Reusable Arm64 E2E Workflow
```

Comment:
Just noticed we should probably align this name with the existing e2e per pull request workflow which is `E2E`: https://github.com/etcd-io/etcd/blob/main/.github/workflows/e2e.yaml#L2

So we could make this `E2E-Arm64`.

Reply:
Make sense.